projects/RabbitMQ.Client/client/api/ConnectionFactory.cs

```diff
@@ -274,6 +274,13 @@ public TimeSpan ContinuationTimeout
         /// </summary>
         public bool TopologyRecoveryEnabled { get; set; } = true;
 
+        /// <summary>
+        /// Force writes to the socket to run on a dedicated thread instead of the thread pool. This may prevent
+        /// timeouts if a large number of blocking requests are going out simultaneously. Will become obsolete
+        /// once requests become asynchronous. Defaults to false.
+        /// </summary>
+        public bool EnableSynchronousWriteLoop { get; set; } = false;
```

Comment:
Since the default value is `false`, maybe we could remove the init value
```suggestion
        public bool EnableSynchronousWriteLoop { get; set; }
```

Reply:
Making the default more visible is good idea. I'd not remove it.

Reply:
Overall, I think it's a kind of code style. I prefer to remove it while I would not argue for this.

Have a test on this, the default value would be removed when built in release mode, would not when in debug mode

![](https://github.com/rabbitmq/rabbitmq-dotnet-client/assets/7604648/35dc298a-9e28-41c7-9eeb-56d3f156bca6)